target/linux/ramips/dts/mt7620a_asus_rp-n53.dts

```diff
@@ -134,6 +134,11 @@
 	};
 };
 
+&ethernet {
+	pinctrl-names = "default";
+	pinctrl-0 = <&ephy_pins>;
+};
```

Comment:
Is this added by accident? If not, it should be in a separated commit.

Reply:
In this DTS, I am removing the `&gsw` node like the others, but this time there is also pinctrl related properties. I am making the `&ethernet` node (which is previously missing) in order to keep these pinctrl related properties from the other node. Almost every other ramips device with these pinctrl properties for ephy have them in the ethernet node, so it is now consistent with other similar DTS.